src/plugins/helpers/storage.ts

```diff
@@ -0,0 +1,120 @@
+import { MMKV } from 'react-native-mmkv';
+
+const store = new MMKV({ id: 'plugin_db' });
+
+const PLUGIN_STORAGE = '_DB_';
+const WEBVIEW_LOCAL_STORAGE = '_LocalStorage';
+const WEBVIEW_SESSION_STORAGE = '_SessionStorage';
+
+interface StoredItem {
+  created: Date;
+  value: any;
+  expires?: Date;
+}
+
+class Storage {
+  /**
+   * Sets a key-value pair in the storage.
+   *
+   * @param pluginID - The ID of the plugin.
+   * @param key - The key to set.
+   * @param value - The value to set.
+   * @param expires - Optional. The expiration date for the key-value pair.
+   */
+  set(
+    pluginID: string,
+    key: string,
+    value: any,
+    expires?: Date | number,
+  ): void {
+    const item: StoredItem = {
+      created: new Date(),
+      value,
+      expires: expires instanceof Date ? expires.getTime() : expires,
+    };
+    store.set(pluginID + PLUGIN_STORAGE + key, JSON.stringify(item));
```

Comment:
for set and get json object, you can reuse the  `getMMKVObject<T>` and `setMMKVObject<T>` in utils/mmkv

Reply:
I am using a different instance of `mmkv`.
```ts
const store = new MMKV({ id: 'plugin_db' });
```

The default mmkv instance is used in `utils/mmkv`.
```ts
export const MMKVStorage = new MMKV();
```

Reply:
I think we should use the default MMKV instance as singleton instead creating a new one. This is also recommended by the author of react-native-mmkv